api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -84,24 +138,39 @@ service ConfigMgmt {
     option (chef.automate.api.iam.policy).resource = "infra:nodes";
     option (chef.automate.api.iam.policy).action = "infra:nodes:list";
   };
+
+  /*
+  GetAttributes
+
+  Returns the latest reported attributes for the provided node id.
+	*/
   rpc GetAttributes (cfgmgmt.request.Node) returns (cfgmgmt.response.NodeAttribute) {
     option (google.api.http).get = "/cfgmgmt/nodes/{node_id}/attribute";
     option (chef.automate.api.policy).resource = "cfgmgmt:nodes:{node_id}:attribute";
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "infra:nodes:{node_id}";
     option (chef.automate.api.iam.policy).action = "infra:nodes:get";
   };
+
   rpc GetVersion(common.version.VersionInfoRequest) returns (common.version.VersionInfo) {
     option (google.api.http).get =  "/cfgmgmt/version";
     option (chef.automate.api.policy).resource = "service_info:version";
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "system:service:version";
     option (chef.automate.api.iam.policy).action = "system:serviceVersion:get";
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "hidden";
+		};
   };
+
+  /*
+  GetPolicyCookbooks
+
+  Returns the policy information for a node.
+  The revision_id ??????
```

Comment:
I would say something like: 
Returns Policy Names with a list of cookbook names and associated policy identifiers based on a policy revision ID. Policy revision ID's are sent with an infra run report.

Reply:
thank you!